source/fundamentals/bson.txt

```diff
@@ -213,21 +212,31 @@ Driver will marshal structs using the following rules:
 BSON Options
 ------------
 
-You can set ``BSONOptions`` to specify how
-the driver marshals and unmarshals BSON on your collection, database, or
-client.  
+You can specify a ``BSONOptions`` instance to set BSON options on your ``Client``.
+BSON options adjust how the driver marshals and unmarshals BSON data.
 
-The following example demonstrates how to set ``BSONOptions`` for 
-the ``movies`` collection:
+This example specifies the following ``BSONOptions`` struct fields:
+
+- ``UseJSONStructTags``, which instructs the driver to use the ``"json"``
+  struct tag if a ``"bson"`` struct tag is not specified
+- ``DefaultDocumentD``, which instructs the driver to always unmarshal documents
+  into the ``primitive.D`` type
+
+The following example uses the ``SetBSONOptions()`` method with a ``BSONOptions``
+parameter to specify a ``ClientOptions`` instance:
 
 .. code-block:: go
-   :copyable: false
 
    bsonOpts := &options.BSONOptions {
        UseJSONStructTags: true,
+       DefaultDocumentD: true,
    }
-   coll := client.Database("sample_mflix")
-       .Collection("movies",bsonOpts)
+
+   clientOpts := options.Client().
+       ApplyURI("mongodb://localhost:27017").
+       SetBSONOptions(bsonOpts)
+
+   client, err := mongo.Connect(context.TODO(), clientOpts)
 
 .. tip::    
```

Comment:
I: this tip does not actually link to the BSONOptions API documentation. Can you rewrite this tip so that it links to the API documentation and the BSONOptions example (making link text clear though eg To learn more about the type, see api documentation. To see an example, <link to example>)